lib/block_view.go

```diff
@@ -7651,6 +7918,7 @@ func (bav *UtxoView) _connectNFTTransfer(
 	newNFTEntry.OwnerPKID = receiverPKID.PKID
 	newNFTEntry.UnlockableText = txMeta.UnlockableText
 	newNFTEntry.IsPending = true
+	newNFTEntry.IsBuyNow = false
```

Comment:
Should we make it so IsBuyNow can only be set after IsPending = false? I don't think it's necessary but just a thought I had while reading.

Reply:
hmm I think we should? I guess it's a little strange to sell a pending NFT lol